src/sockets/sockets.ts

```diff
@@ -2088,59 +2090,67 @@ function kickPlayer(username: string): void {
   }
 }
 
-function joinQueue() {
+function queueRequest(): void {
   const username = this.request.user.username;
-  const blacklistUsernames = this.request.user.matchmakingBlacklist;
 
-  if (!joinQueueFilter.joinQueueRequest(username)) {
-    this.emit('allChatToClient', {
-      message:
-        'You have rejected too many found matches. Please try again later.',
-      classStr: 'server-text',
-    });
-    return;
-  }
+  if (this.data.joined) {
```

Comment:
This shouldn't be `this.data.joined`. The data should be on line 2093 in the queueRequest params.